danger-klaxit/lib/danger_plugin.rb

```diff
@@ -119,6 +120,29 @@ def warn_for_bad_order_in_config(config_file = "config/config.yml")
     MARKDOWN
   end
 
+  def warn_for_not_updated_structure_sql
+    migration_files = (git.added_files - %w(Dangerfile)).grep(%r(db/migrate))
+    return nil if migration_files.empty?
+
+    if git.modified_files.grep(%r(db/structure.sql)).empty?
```

Comment:
What about `schema.rb`?
```suggestion
    if git.modified_files.grep(%r(db/(?:structure\.sql|schema\.rb))).empty?
```

Idem for the message!